wscript

```diff
@@ -6,13 +6,16 @@ sys.path.insert(0, os.getcwd())
 from shlex import split
 from waflib.Configure import conf
 from waflib.Tools import c_preproc
+from waflib.Tools.compiler_c import c_compiler
 from waflib import Utils
 from waftools.checks.generic import *
 from waftools.checks.custom import *
 
 c_preproc.go_absolute=True # enable system folders
 c_preproc.standard_includes.append('/usr/local/include')
 
+c_compiler['win32'].remove('msvc')
```

Comment:
another one sorry: I don't see this being explained anywhere

Reply:
I think mpv cannot be built with MSVC, no? For sure it is not supported case. So I removed form compiler lookup.

To be more specific about the change. This is another stupidness of waf. It has lists of compilers per each platform see https://waf.io/apidocs/tools/compiler_c.html#waflib.Tools.compiler_c.c_compiler And now even if you set CC to proper compiler it will happily still (try to) use cl.exe (msvc) if you have it in path. I don't know who on earth though it is good idea... You can do `--check-c-compiler=gcc` but since we don't support msvc anyway, I removed it form autodetection.

Reply:
Ah sure. Please split this into a separate commit.

Reply:
ok